dashboard/config/locales/en.yml

```diff
@@ -170,6 +170,8 @@ en:
     invalid_password: "Password must be at least 6 characters"
     mismatch_password: "The two passwords you entered do not match"
     student_consent: "I confirm that I have my parent or legal guardian's permission to use the Code.org services."
+    agree_us_website: "I agree to using a website based in the United States."
```

Comment:
Some of these changes don't look related to your change.  Maybe there's some merge funniness?

Reply:
Yep - not exactly sure what happened. Apologies for not noticing this before opening the PR. I put my questionable git skills to use to resolve the issue so hopefully it is fixed the next time i push.